.circleci/config.yml

```diff
@@ -12,12 +12,12 @@ install-deps: &install-deps
 executors:
   node-container-lts:
     docker:
-      - image: node:16
+      - image: cimg/node:16.13
```

Comment:
16.14.0 is the lts, no? https://circleci.com/developer/images/image/cimg/node

Reply:
Correct, updated 👍